src/http/mod.rs

```diff
@@ -70,11 +70,16 @@ pub fn min_ttl<'a>(res_vec: impl Iterator<Item = &'a Response>) -> i32 {
   min
 }
 
-fn log_launch(sc: &ServerConfig) {
+fn log_launch_and_open_browser(sc: &ServerConfig) {
   let addr = sc.addr().to_string();
   log::info!("🚀 Tailcall launched at [{}] over {}", addr, sc.http_version());
   if sc.graphiql() {
-    log::info!("🌍 Playground: {}", sc.graphiql_url());
+    let url = sc.graphiql_url();
+    log::info!("🌍 Playground: {}", url);
+
+    if let Ok(_) = webbrowser::open(url.as_str()) {
+      log::info!("{} opened on your browser", url);
+    }
```

Comment:
No need for these logs actually. Just open silently and let it fail also silently.

Reply:
done